doc/bql.rst

```diff
@@ -206,7 +206,7 @@ variables in a population.
 
    Drop the metamodel *metamodel* and all its contents.
 
-.. index:: ``INITIALIZE MODELS``
+.. index:: ``INITIALIZE``
```

Comment:
It was intentional --- I realized that `INITIALIZE` without `MODELS` was more consistent with the existing style. I was not using `git add -p` but will in the future.